deploy/common/k8sbased.sh

```diff
@@ -200,7 +200,7 @@ function launch_central {
     fi
 
     # TODO(ROX-16008): Once the feature flag is enabled by default, always add the config map mount.
-    if [[ -n "${ROX_DECLARATIVE_CONFIGURATION}" ]]; then
+    if [[ -n "${ROX_DECLARATIVE_CONFIGURATION}" && "${IS_RELEASE_BUILD}" != "true" ]]; then
```

Comment:
@gavin-stackrox alternatively instead of using `export_test_environment` we could create a function within the `k8sbased.sh` as such (which is essentially a duplication of the functions in the script so _meh_):
```bash
RELEASE_RC_TAG_BASH_REGEX='^([[:digit:]]+(\.[[:digit:]]+)*)(-rc\.[[:digit:]]+)?$'

function is_release_build {
    [[ "${MAIN_IMAGE_TAG}" =~ $RELEASE_RC_TAG_BASH_REGEX && -z "${BASH_REMATCH[3]}" ]] || [[ "${BUILD_TAG:-}" =~ -nightly- ]] || [[ "${GITHUB_REF:-}" =~ nightly- ]]
}
```